src/v/archival/archiver_operations_impl.cc

```diff
@@ -41,6 +41,33 @@ namespace archival {
 namespace detail {
 
 class archiver_operations_impl : public archiver_operations_api {
+    /// Combined result of several uploads
+    struct aggregated_upload_result {
```

Comment:
Should this be in this commit? It doesn't seem used

Reply:
moved to another commit (git absorb is to blame)